tests/gnrc_tcp_client/Makefile

```diff
@@ -1,10 +1,13 @@
 include ../Makefile.tests_common
 
 # If no BOARD is found in the environment, use this default:
-BOARD ?= native
+BOARD = native
```

Comment:
Please remove that change. Otherwise you can't overwrite the board variable. How did you test this way?

Reply:
I'll revert the makefile changes